tests/test_modeling_bart.py

```diff
@@ -109,7 +110,9 @@ class BARTModelTest(ModelTesterMixin, unittest.TestCase):
     all_model_classes = (
         (BartModel, BartForConditionalGeneration, BartForSequenceClassification) if is_torch_available() else ()
     )
-    all_generative_model_classes = (BartForConditionalGeneration,) if is_torch_available() else ()
+    all_generative_model_classes = (
+        (BartForConditionalGeneration, MarianForConditionalGeneration) if is_torch_available() else ()
```

Comment:
very cheap test coverage: + 2 seconds. Happy to delete.

Reply:
I think it's good to have this.